dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/report/MetadataReport.java

```diff
@@ -45,7 +45,7 @@
     default void publishAppMetadata(SubscriberMetadataIdentifier identifier, MetadataInfo metadataInfo) {
     }
 
-    default MetadataInfo getAppMetadata(SubscriberMetadataIdentifier identifier, Map<String, String> instanceMetadata) {
+    default MetadataInfo getAppMetadata(SubscriberMetadataIdentifier identifier) {
```

Comment:
Please do not change this method signature. `instanceMetadata` contains some instance releated properties which will take effect when getting metadata